vassal-app/src/main/java/VASSAL/counters/GlobalCommand.java

```diff
@@ -127,7 +127,9 @@ public GlobalCommandTarget getTarget() {
    */
   private boolean passesPropertyFastMatch(GamePiece gamePiece) {
     if (!target.fastMatchProperty || fastProperty.isEmpty()) return true;
-    final String value = gamePiece.getProperty(fastProperty).toString();
+
+    final Object prop = gamePiece.getProperty(fastProperty);
+    final String value = (prop == null) ? "" : prop.toString();
```

Comment:
Can't you let null just be null instead of the empty string here, since you check for null below?